src/components/Profile/Profile.css

```diff
@@ -0,0 +1,106 @@
+* {
```

Comment:
As we have discussed, we will move forward with scss, so you shouldn't mix plain css in our project.